recipes/neon2sse/all/conanfile.py

```diff
@@ -0,0 +1,55 @@
+from conans import ConanFile, tools, CMake
+from conans.errors import ConanInvalidConfiguration
+import os
+
+
+required_conan_version = ">=1.33.0"
+
+class Neon2sseConan(ConanFile):
+    name = "neon2sse"
+    url = "https://github.com/conan-io/conan-center-index"
+    homepage = "https://github.com/intel/ARM_NEON_2_x86_SSE"
+    description = "Header only library intended to simplify ARM->IA32 porting"
+    license = "BSD-2-Clause"
+    topics = "neon", "sse", "port", "translation", "intrinsics"
+    settings = "os", "compiler", "build_type", "arch"
+    generators = "cmake"
+    exports_sources = "CMakeLists.txt"
+
+    _cmake = None
+
+    @property
+    def _source_subfolder(self):
+        return "source_subfolder"
+
+    def _configure_cmake(self):
+        if self._cmake:
+            return self._cmake
+        self._cmake = CMake(self)
+        self._cmake.configure()
+        return self._cmake
+    
+    def validate(self):
+        if self.settings.arch not in ("x86", "x86_64"):
+            raise ConanInvalidConfiguration("neon2sse only supports arch={x86,x86_64}")
+
+    def source(self):
+        tools.get(**self.conan_data["sources"][self.version],
+            destination=self._source_subfolder, strip_root=True)
+
+    def build(self):
+        cmake = self._configure_cmake()
+        cmake.build()
+
+    def package(self):
+        cmake = self._configure_cmake()
+        cmake.install()
+        self.copy("LICENSE", dst="licenses", src=self._source_subfolder)
+        tools.rmdir(os.path.join(self.package_folder, "lib"))
+
+    def package_id(self):
+        self.info.header_only()
+
+    def package_info(self):
```

Comment:
Not really needed, but this avoid warnings emitted by the hook:
```
neon2sse/cci.20210225: WARN: Lib folder doesn't exist, can't collect libraries: /home/conan/w/BuildSingleReference/.conan/data/neon2sse/cci.20210225/_/_/package/5ab84d6acfe1f23c4fae0ab88f26e3a396351ac9/lib
```
(see the logs of the test package)
```suggestion
    def package_info(self):
        self.cpp_info.libdirs = []
```

Reply:
I'm all in when it comes to suppressing warnings :100: